src/styles/main.scss

```
@@ -1,3 +1,72 @@
-body {
-  margin: 0;
+$calendar-gap: 1px;
+$cell-size: 100px;
+$calendar-width: 6 * $calendar-gap + 7 * $cell-size;
+$days:
+  "mon" 0,
+  "tue" $cell-size + $calendar-gap,
+  "wed" 2 * ($cell-size + $calendar-gap),
+  "thu" 3 * ($cell-size + $calendar-gap),
+  "fri" 4 * ($cell-size + $calendar-gap),
+  "sat" 5 * ($cell-size + $calendar-gap),
+  "sun" 6 * ($cell-size + $calendar-gap);
+$month-length: 28, 29, 30, 31;
+
+* {
+  box-sizing: border-box;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-gap;
+
+  margin: 50vh auto;
+  max-width: $calendar-width;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: $cell-size;
+    height: $cell-size;
+
+    background-color: #eee;
+    border: 1px solid #000;
+    transition: background-color 0.5s, transform 0.5s;
```

don't forget about vars
```suggestion
    border: 1px solid $c-black;
    transition: background-color $transitionDuration, transform $transitionDuration;
```